.github/scripts/upload-scribe.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# [FB Internal] Upload every json file to Scribe
+
+set -eo pipefail
+
+if [[ ! -v SCRIBE_GRAPHQL_ACCESS_TOKEN ]]; then
+    echo "SCRIBE_GRAPHQL_ACCESS_TOKEN is not set! Please check your environment variable. Stop."
+    exit 1
+fi
+
+pushd /workspace/benchmark/score
+
+# Generate the benchmark config
+python generate_score_config.py --specification score.yml \
```

Comment:
this doesn't look right to me. we shouldn't be generating a new score config on every night.  if we do, won't we just get a score of 'target_score' every night?

Reply:
fwiw, the score scripts are not user friendly.. but what i had in mind by designing this way was 
 - one time, admin would set up 'golden machine' and create score config on it
   - check this score config in, with a version number attached to its name
- then, anyone wanting to compute the score would use that file, use the same machine (or at least same type of machine), and run the same benchmarks.

Reply:
That is correct. We need to fix this problem by first determining the 'golden machine' and the config that it uses.

Reply:
ok. as long as you have a plan to deal with that.